composer.json

```diff
@@ -15,7 +15,7 @@
     }
   ],
   "require": {
-    "imagekit/imagekit": "^3.0.0"
+    "imagekit/imagekit": "^4.0"
```

Comment:
The ImageKit SDK is used to just generate the URL's, the changelog between v3 and v4 is mostly regarding deprecating the overlay syntax and upload, so everything will work fine without any problem